Changelog.md

```diff
@@ -0,0 +1,29 @@
+# Recent Changes
+
+* Fix: Astro Integrations support  
+  (add missing `package.json` keywords and exports)
+
+* Declarative Shadow DOM polyfill improvements
+  
+  - More efficient DSD feature detection and polyfill
+    (based on: <https://developer.chrome.com/articles/declarative-shadow-dom>)
+
+  - Polyfill injection now happens only if pages actually use DSD
+  
+  - Fix: Check if Page isn't already loaded before installing the polyfill handler.
+
+* Fix: Support Custom Elements without shadow DOM
+
+* Fix: Slot handling
+
+* Feature: e2e Tests
+
+## TODO
+
+* Render and hydration hook
+
+* Lit and Rust Test examples
```

Comment:
Rust test examples?

Reply:
Sounds cool, id love to learn more about this, do you have any links to share on this by any chance?

Reply:
Hmm... unfortunately there isn't much useful to link about my desires concerning this topic.

As base for my experiments with rust based web components, I utilized mostly [silkenweb](https://github.com/silkenweb/silkenweb). That's a less popular rust frontend framework, but it's very well maintained, not bloated with useless features and provides a very clean and efficient hydration solution for components out of the box.

Until now there is IMHO no rust SSG solution available, which could be seen as a sufficient powerful and flexible replacement for Astro and similar JS based tools and metaframeworks of the last generation.